proto/streamlit/proto/ForwardMsg.proto

```diff
@@ -58,10 +58,13 @@ message ForwardMsg {
     GitInfo git_info_changed = 14;
     PageProfile page_profile = 18;
 
-    // State change and event messages.
-    SessionState session_state_changed = 9;
+    // Status change and event messages.
+    SessionStatus session_status_changed = 19;
     SessionEvent session_event = 10;
 
+    // DEPRECATED
+    // SessionState session_state_changed = 9;
+
```

Comment:
Is this change necessary? Since `SessionStatus` is binary-compatible with `SessionState`, changing the message IDs shouldn't be necessary. Any protobuf with a `SessionState` should look identical to one that has a `SessionStatus` in the same place, right?

Reply:
Ah whoops -- didn't realize that renames were safe. If that's the case, will just keep the tag numbers the same